package.json

```diff
@@ -1192,15 +1065,6 @@
 					"getProjectSetupInfo",
 					"installExtension"
 				]
-			},
-			{
-				"name": "runTasks",
```

Comment:
Is this registered from core now?

Reply:
yes

Reply:
https://github.com/microsoft/vscode/blob/7a614035a9aa8c0a74c225ee04c180e767fd07a7/src/vs/workbench/contrib/terminalContrib/chatAgentTools/browser/task.chatAgentTools.contribution.ts#L40-L42

Reply:
looks like I need to rename that `runTasks`, will do.

Reply:
https://github.com/microsoft/vscode/pull/257730